notebooks/byod.py

```diff
@@ -183,16 +148,16 @@ def create_cram_crai_table(table: str, listing: Iterable[str]):
     ### Add a prefix to your bucket path to organize your data
     In this example, we add the prefix 'test-crai-cram'. In the terminal of your computer, you will call something like:
 
-    `gsutil cp /Users/Documents/Example.cram gs://your_bucket_info/test-crai-cram/`
-    """
+    `gsutil cp /Users/my-cool-username/Documents/Example.cram gs://your_bucket_info/my-crams/`
+
+    We will be calling what comes after your bucket info, here represented as `my-crams`, as your sudirectory.
 
-with callysto.Cell("markdown"):
-    """
     ## Preview the data in your workspace bucket
+    Be aware that if you have uploaded multiple files, all of them will appear with this ls command. It will also contain one folder for every workflow you have run in this workspace. You may want to skip this step if you're after uploading hundreds of files. However, if you have imported data tables from Gen3, they will not show up here as the files within are only downloaded when their associated TSV tables are called upon by workflows or iPython notebooks.
```

Comment:
Users should be encouraged to explore the contents of their workspace bucket, not discouraged. Also, large output is scrollable in Jupyter cells. Even long listings will not overwhelm the screen.

Please consider removing information extraneous to this notebook, such as discussion about workflow folders and Gen3 tables.

Reply:
I find the scrolling on Terra notebooks to be a bit unresponsive (at least for me -- I zoom almost all webpages in by 10-20%, which can break internal scrolls sometimes), so I shy away from it. But you make a good point -- let's have them explore. I'll reword this as less of a warning and more of an explanation.